.travis.yml

```diff
@@ -83,3 +83,7 @@ script:
 after_script:
     - echo $TRAVIS_COMMIT_RANGE
     - echo $TRAVIS_COMMIT_LOG
+after_failure:
+    - pip3 install irc --user
+    - if [[ "$TRAVIS_REPO_SLUG" = "bitcoin/bitcoin" && ( "$TRAVIS_EVENT_TYPE" = "push" || "$TRAVIS_EVENT_TYPE" = "cron" ) ]]; then wget "https://raw.githubusercontent.com/jaraco/irc/60f747a578157b03c952c050c765e2924d4fdc00/scripts/irccat2.py"; fi
```

Comment:
Repeating this condition is a bit ugly. As this is YAML, isn't it possible to spread a script over multiple lines?

Reply:
Jup, that is possible